### ISSUE TITLE
Pasting a multiline string from the clipboard causes the message buffer to automatically submit to the server

### ISSUE THREAD

**Issue by gbakeman** [icechat/IceChat#35]
IceChat 9.53

I've just accidentally discovered that if I paste a multiline string into my chat buffer, it will automatically be sent to the server. It would be nice new line characters are ignored or escaped to help prevent these unintentional submissions.

Thanks for your work on this!

**Comment by icechat** [comment]
Use the Multiline Edit Box to prevent this. It is an option under the View Menu.

**Comment by gbakeman** [comment]
Fair enough, I'll check it out. I was hoping that default behavior of the single line box would be something like notifying the user of the problem (by way of a system sound, notification bubble, etc.) or just accept the text input and strip out the newlines. It was somewhat unexpected (and inconveinent for the channel members) for the accidental multiline paste to go straight to the server. Would it make sense to change this default behavior?

**Comment by icechat** [comment]
I do not think so. This is why the multiline edit box was made as an option.
